### PR TITLE
Closes #2197 use card title field in demo content, Closes #2426 Add view paragraphs to demo content.

### LINKED ISSUE #2197: Update Demo content to utilize Card Title fields
Our current demo content for Cards use Text fields to provide titles for each Card group. 
Once #2192 is merged we will want to update the demo content to use the new Card group title on this page: /pages/card-decks

## Conditions of satisfaction
- [ ] Text areas are no longer used for card group titles
- [ ] Each card group has an appropriate title

### LINKED ISSUE #2426: Add new AZ Person > Grid 3-col block page to Demo Content
## Motivation
I just completed [a PR](https://github.com/az-digital/az_quickstart/pull/2381) to add a 3-column view display to AZ Person. Suggested we also make it available in the Person menu item in Demo Content.

### Describe the solution you'd like
The default AZ Person Grid Page remains and is 4 columns. This just adds a block to the view display so someone can add a new page for 3 columns.

### Describe alternatives you've considered
Block was chosen over Page for reasons [Joe outlined in the PR](https://github.com/az-digital/az_quickstart/pull/2381#pullrequestreview-1434812147).

### PR DESCRIPTION
**Pull request by trackleft**
## Description
Added new page, and menu item for news views paragraphs.
Added 3 new paragraphs with view references
Update card deck migration to use field_az_title, which was added to quickstart.

## Related issues
Closes #2197 
Closes #2426 

## How to test
Go to /pages/news-view-paragraphs
See views embedded in a page via views paragraph

See https://8d3d666c-b5e0-4a90-95cc-308918134ee6.probo.build/pages/card-decks
See https://8d3d666c-b5e0-4a90-95cc-308918134ee6.probo.build/pages/news-view-paragraphs
See https://8d3d666c-b5e0-4a90-95cc-308918134ee6.probo.build/pages/person-view-paragraphs

Existing card deck page https://demo.quickstart.arizona.edu/pages/card-decks

Go to /pages/card-decks
And see new card deck group titles

### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
   - [ ] Bug fix
   - [ ] Accessibility, performance, or security improvement
   - [ ] Critical institutional link or brand change
   - [ ] Adding experimental module
   - [ ] Update experimental module
- **Minor release changes**
   - [x] New feature
   - [ ] Breaking or visual change to existing behavior
   - [ ] Upgrade experimental module to stable
   - [ ] Enable existing module by default or database update
   - [ ] Non-critical brand change
   - [ ] New internal API or API improvement with backwards compatibility
   - [ ] Risky or disruptive cleanup to comply with coding standards
   - [ ] High-risk or disruptive change (requires upgrade path, risks regression, etc.)
- **Other or unknown**
   - [ ] Other or unknown

### Drupal core
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch level release (non-security bug-fix release)
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major or minor level update
- **Other or unknown**
   - [ ] Other or unknown

### Drupal contrib projects
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch or minor level update
   - [ ] Add new module
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major level update
- **Other or unknown**
   - [ ] Other or unknown

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by trackleft** [comment]
TODO:

- [ ] Remove extra text areas that are no longer required on the card decks page.

File: modules/custom/az_demo/data/az_demo_page_node.json [hunk 1]
```diff
@@ -180,6 +180,26 @@
...
+      "path" : "/pages/news-view-paragraphs",
+      "exclude_title" : true,
+      "az_page_category" : "Views",
+      "az_main_content" : "100|101|102",
+      "az_media_image" : "",
+      "az_summary" : "A flexible page with news views."
+    },
+    {
+      "id":20,
+      "title" : "Person Grid - 3 Col",
```
Comment (trackleft) [inline modules/custom/az_demo/data/az_demo_page_node.json:196]:
```suggestion
      "title" : "Person Views",
```

File: modules/custom/az_demo/data/az_demo_page_node.json [hunk 2]
```diff
@@ -180,6 +180,26 @@
...
+      "path" : "/pages/person-grid-3-col",
```
Comment (trackleft) [inline modules/custom/az_demo/data/az_demo_page_node.json:197]:
```suggestion
      "path" : "/pages/person-view-paragraphs",
```

File: modules/custom/az_demo/data/az_demo_page_node.json [hunk 3]
```diff
@@ -180,6 +180,26 @@
...
+      "exclude_title" : true,
+      "az_page_category" : "Views",
+      "az_main_content" : "59",
```
Comment (trackleft) [inline modules/custom/az_demo/data/az_demo_page_node.json:200]:
```suggestion
      "az_main_content" : "103",
```

File: modules/custom/az_demo/data/az_demo_view_paragraph.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  "paragraphs": [
+    {
+      "id":100,
+      "target_id" : "az_news",
+      "display_id" : "az_grid",
+      "data": "",
+      "bottom_spacing" : "mb-0"
+    },
+    {
+      "id":101,
```
Comment (trackleft) [inline modules/custom/az_demo/data/az_demo_view_paragraph.json:11]:
```suggestion
      "id":101,
       "field_az_title" : "Grid View with Filter",

```

File: modules/custom/az_demo/migrations/az_demo_view_paragraph.yml [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    -
+      name: bottom_spacing
+      selector: bottom_spacing
+
+destination:
+  plugin: 'entity_reference_revisions:paragraph'
+  default_bundle: az_view_reference
+
+process:
+  field_az_view_reference/target_id: target_id
```
Comment (trackleft) [inline modules/custom/az_demo/migrations/az_demo_view_paragraph.yml:44]:
```suggestion
  field_az_title: field_az_title
```

File: modules/custom/az_demo/migrations/az_demo_view_paragraph.yml [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+    -
+      name: bottom_spacing
+      selector: bottom_spacing
+
+destination:
+  plugin: 'entity_reference_revisions:paragraph'
+  default_bundle: az_view_reference
+
+process:
+  field_az_title/0/value: field_az_title
```
Comment (trackleft) [inline modules/custom/az_demo/migrations/az_demo_view_paragraph.yml:47]:
```suggestion
  field_az_title/value: field_az_title
```

File: modules/custom/az_demo/migrations/az_demo_view_paragraph.yml [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+  ids:
+    id:
+      type: integer
+  fields:
+    -
+      name: id
+      selector: id
+    -
+      name: field_az_title
+      selector: field_az_title
```
Comment (trackleft) [inline modules/custom/az_demo/migrations/az_demo_view_paragraph.yml:28]:
```suggestion
    -
      name: title
      selector: title
```

File: modules/custom/az_demo/data/az_demo_view_paragraph.json [hunk 8]
```diff
@@ -0,0 +1,36 @@
+{
+  "paragraphs": [
+    {
+      "field_az_title" : "Grid View",
```
Comment (trackleft) [inline modules/custom/az_demo/data/az_demo_view_paragraph.json:4]:
```suggestion
      "title" : "Grid View",
```

File: modules/custom/az_demo/data/az_demo_view_paragraph.json [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+      "id":100,
+      "target_id" : "az_news",
+      "display_id" : "az_grid",
+      "data": "",
+      "bottom_spacing" : "mb-0"
+    },
+    {
+      "id":101,
+      "field_az_title" : "Grid View with Filter",
```
Comment (trackleft) [inline modules/custom/az_demo/data/az_demo_view_paragraph.json:13]:
```suggestion
      "title" : "Grid View with Filter",
```

File: modules/custom/az_demo/data/az_demo_view_paragraph.json [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+      "target_id" : "az_news",
+      "display_id" : "az_grid_filter",
+      "data": "",
+      "bottom_spacing" : "mb-0"
+    },
+    {
+      "id":102,
+      "field_az_title" : "Feature View",
```
Comment (trackleft) [inline modules/custom/az_demo/data/az_demo_view_paragraph.json:21]:
```suggestion
      "title" : "Feature View",
```

File: modules/custom/az_demo/data/az_demo_view_paragraph.json [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+      "target_id" : "az_news",
+      "display_id" : "az_feature",
+      "data": "",
+      "bottom_spacing" : "mb-0"
+    },
+    {
+      "id" : 103,
+      "field_az_title" : "Grid - 3 col",
```
Comment (trackleft) [inline modules/custom/az_demo/data/az_demo_view_paragraph.json:29]:
```suggestion
      "title" : "Grid - 3 col",
```

File: modules/custom/az_demo/migrations/az_demo_view_paragraph.yml [hunk 12]
```diff
@@ -0,0 +1,55 @@
...
+    -
+      name: bottom_spacing
+      selector: bottom_spacing
+
+destination:
+  plugin: 'entity_reference_revisions:paragraph'
+  default_bundle: az_view_reference
+
+process:
+  field_az_title/value: field_az_title
```
Comment (trackleft) [inline modules/custom/az_demo/migrations/az_demo_view_paragraph.yml:47]:
```suggestion
  field_az_title/value: title
```

File: modules/custom/az_demo/migrations/az_demo_view_paragraph.yml [hunk 13]
```diff
@@ -0,0 +1,55 @@
...
+    -
+      name: bottom_spacing
+      selector: bottom_spacing
+
+destination:
+  plugin: 'entity_reference_revisions:paragraph'
+  default_bundle: az_view_reference
+
+process:
+  field_az_title/value: title
```
Comment (trackleft) [inline modules/custom/az_demo/migrations/az_demo_view_paragraph.yml:47]:
```suggestion
  field_az_title/value: 
  - plugin: skip_on_empty
     method: process
     source: title
```

**Review by joeparsons — APPROVED** [review]

**Review by sanjanans — APPROVED** [review]

**Pull request merged**